scripts/chat.py

```diff
@@ -63,10 +64,9 @@ def chat_with_ai(
             """
             model = cfg.fast_llm_model # TODO: Change model from hardcode to argument
             # Reserve 1000 tokens for the response
-            
-            if cfg.debug:
+            if os.environ.get("DEBUG"):
```

Comment:
I think it's better to keep using `cfg.debug` and then move the env var to the `Config.debug` code. Or call `cfg.set_debug_mode(os.environ.get("DEBUG"))` even though those setter/getters are not pythonic.